apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/StandardCacheHeaders.java

```diff
@@ -0,0 +1,14 @@
+package com.apollographql.apollo.cache.normalized;
+
+/**
+ * A collection of "standardized" cache headers that an implementation {@link NormalizedCache}
+ * is recommended to respect.
+ */
+public class StandardCacheHeaders {
+
+  /**
+   * Records from this request should not be stored in the {@link NormalizedCache}.
+   * This does **not** specify that a request should not be read from a cache.
+   */
+  public static final String DO_NOT_CACHE = "do-not-cache";
```

Comment:
I can merge 'CacheHeaders` and `CacheControl` into a single object if that seems better. But they are different concepts.

 I would argue that `CacheControl` should have been named `FetchStrategy` -- it controls how we look up data.

`CacheHeaders` instructs the cache to do certain things -- such as not storing records, or returning null because the TTL is expired.